### PR TITLE
Switch analysis input

### PR DESCRIPTION
**Pull request by ghost**
Scenario_data is now used directly from data repo.
Change check for bonds passed.
I sneakily added a small fix in the bonds_change_check also.

### CODE REVIEW

File: R/read_scenario_data.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#' Read in scenario data
+#'
+#' Function reads in scenario data and checks for existence of required columns.
+#'
+#' @param path Path to dir holding scenario data.
+#'
+#' @return A tibble holding scenario data.
+read_scenario_data <- function(path) {
+  scenario_data <- readr::read_csv(path, col_types = "ccccccncn")
```
Comment (jacobvjk) [inline R/read_scenario_data.R:9]: let's make the col_types explicit while we are at it

File: R/wrangle_and_check.R [hunk 2]
```diff
@@ -126,3 +125,24 @@ wrangle_financial_data <- function(financial_data, start_year) {
...
+#'
+#' @param scenario_data A tibble holding scenario data.
+#' @param start_year Start year of analysis.
+#' @param end_year End year of analysis.
+#'
+#' @return A tibble holding wrangled scenario_data.
+wrangle_scenario_data <- function(scenario_data, start_year, end_year) {
+  scenario_data_wrangled <- scenario_data %>%
+    dplyr::rename(source = .data$scenario_source) %>%
+    dplyr::filter(.data$source %in% c("ETP2017", "WEO2019")) %>%
```
Comment (jacobvjk) [inline R/wrangle_and_check.R:141]: we need to remember to parameterise these scenarios at some point

**Review by jacobvjk — COMMENTED** [review]
see comments

File: R/wrangle_and_check.R (hunk 2 above)
Comment (MirijaHa) [reply R/wrangle_and_check.R:141]: we sure do

File: R/read_scenario_data.R (hunk 1 above)
Comment (MirijaHa) [reply R/read_scenario_data.R:9]: what do you mean by that?

**Pull request merged**